model/src/main/kotlin/FileFormat.kt

```diff
@@ -64,6 +64,16 @@ enum class FileFormat(val mapper: ObjectMapper, val fileExtension: String, varar
          * Return the [FileFormat] for the given [file], or `null` if there is none.
          */
         fun forFile(file: File): FileFormat = forExtension(file.extension)
+
```

Comment:
commit: What's the reason for extracting this function?

Reply:
I'm not certain how we do it for `ort.yml` and other config files.
However, would it make sense to limit the support to just `yml`?

Reply:
We're planning to make reuse of it in an upcoming PR for other YAML / JSON files in a directory tree.

Reply:
> However, would it make sense to limit the support to just `yml`?

I don't think so. While unusual and not really documented, we actually would support `json` files anywhere where we usually use `yml` files, so I'd like to keep the function generic.